
01:56:36: Data gathering run started.
01:56:36: Update flag set to 0

01:57:22: Data gathering run started.
01:57:22: Update flag set to 0
01:57:37: Data from today already gathered. Validating.
01:57:38: The number of cards for date ID 22 is incorrect
01:57:38: Expected: 264    got: 0
01:57:38: Dataset invalid. Proceeding to run.
01:57:38: Update flag set to 1
03:16:40: Update flag set to 0
03:16:41: Message: Unable to execute request for an existing session: java.util.concurrent.TimeoutException
Build info: version: '4.0.0', revision: '3a21814679'
System info: host: '6bf5b61b33e0', ip: '172.31.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.16.3-microsoft-standard-WSL2', java.version: '11.0.11'
Driver info: driver.version: unknown
Stacktrace:
java.lang.RuntimeException: Unable to execute request for an existing session: java.util.concurrent.TimeoutException
Build info: version: '4.0.0', revision: '3a21814679'
System info: host: '6bf5b61b33e0', ip: '172.31.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.16.3-microsoft-standard-WSL2', java.version: '11.0.11'
Driver info: driver.version: unknown
	at org.openqa.selenium.grid.router.HandleSession.execute(HandleSession.java:139)
	at org.openqa.selenium.remote.http.Route$PredicatedRoute.handle(Route.java:373)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.remote.http.Route$CombinedRoute.handle(Route.java:336)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.grid.router.Router.execute(Router.java:91)
	at org.openqa.selenium.grid.web.EnsureSpecCompliantResponseHeaders.lambda$apply$0(EnsureSpecCompliantResponseHeaders.java:34)
	at org.openqa.selenium.remote.http.Filter$1.execute(Filter.java:64)
	at org.openqa.selenium.remote.http.Route$CombinedRoute.handle(Route.java:336)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.remote.http.Route$NestedRoute.handle(Route.java:270)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.remote.http.Route$CombinedRoute.handle(Route.java:336)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.remote.http.Route$CombinedRoute.handle(Route.java:336)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.remote.AddWebDriverSpecHeaders.lambda$apply$0(AddWebDriverSpecHeaders.java:35)
	at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0(ErrorFilter.java:44)
	at org.openqa.selenium.remote.http.Filter$1.execute(Filter.java:64)
	at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0(ErrorFilter.java:44)
	at org.openqa.selenium.remote.http.Filter$1.execute(Filter.java:64)
	at org.openqa.selenium.netty.server.SeleniumHandler.lambda$channelRead0$0(SeleniumHandler.java:44)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.util.concurrent.TimeoutException
	at java.base/java.util.concurrent.CompletableFuture.timedGet(CompletableFuture.java:1886)
	at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2021)
	at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
	at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:66)
	at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
	at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:51)
	at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
	at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:119)
	at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
	at org.openqa.selenium.grid.web.ReverseProxyHandler.execute(ReverseProxyHandler.java:92)
	at org.openqa.selenium.grid.node.ProtocolConvertingSession.execute(ProtocolConvertingSession.java:75)
	at org.openqa.selenium.grid.node.local.SessionSlot.execute(SessionSlot.java:123)
	at org.openqa.selenium.grid.node.local.LocalNode.executeWebDriverCommand(LocalNode.java:388)
	at org.openqa.selenium.grid.node.ForwardWebDriverCommand.execute(ForwardWebDriverCommand.java:35)
	at org.openqa.selenium.remote.http.Route$PredicatedRoute.handle(Route.java:373)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.remote.tracing.SpanWrappedHttpHandler.execute(SpanWrappedHttpHandler.java:86)
	at org.openqa.selenium.remote.http.Filter$1.execute(Filter.java:64)
	at org.openqa.selenium.remote.http.Route$CombinedRoute.handle(Route.java:336)
	at org.openqa.selenium.remote.http.Route.execute(Route.java:68)
	at org.openqa.selenium.grid.node.Node.execute(Node.java:240)
	at org.openqa.selenium.grid.web.CombinedHandler.execute(CombinedHandler.java:59)
	at org.openqa.selenium.grid.web.RoutableHttpClientFactory$1.execute(RoutableHttpClientFactory.java:72)
	at org.openqa.selenium.grid.web.ReverseProxyHandler.execute(ReverseProxyHandler.java:92)
	at org.openqa.selenium.grid.router.HandleSession.execute(HandleSession.java:110)
	... 26 more


03:16:50: Data gathering run started.
03:16:50: Update flag set to 0
03:17:05: Data from today already gathered. Validating.
03:17:06: The number of cards for date ID 22 is incorrect
03:17:06: Expected: 264    got: 169
03:17:06: Dataset invalid. Proceeding to run.
03:17:06: Update flag set to 1
06:21:15: Update flag set to 0
06:21:15: Message: Reached error page: about:neterror?e=connectionFailure&u=https%3A//www.cardmarket.com/en/Magic/Products/Singles/Battlebond/Borderland-Marauder&c=UTF-8&d=Firefox%20can%E2%80%99t%20establish%20a%20connection%20to%20the%20server%20at%20www.cardmarket.com.
Stacktrace:
WebDriverError@chrome://remote/content/shared/webdriver/Errors.jsm:181:5
UnknownError@chrome://remote/content/shared/webdriver/Errors.jsm:488:5
checkReadyState@chrome://remote/content/marionette/navigate.js:64:24
onNavigation@chrome://remote/content/marionette/navigate.js:312:39
emit@resource://gre/modules/EventEmitter.jsm:160:20
receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.jsm:42:25


06:21:18: Data gathering run started.
06:21:18: Update flag set to 0
06:21:33: Data from today already gathered. Validating.
06:21:35: The number of cards for date ID 22 is incorrect
06:21:35: Expected: 264    got: 250
06:21:35: Dataset invalid. Proceeding to run.
06:21:35: Update flag set to 1
06:24:05: Update flag set to 0
06:24:05: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

06:25:47: Data gathering run started.
06:25:47: Update flag set to 0
06:26:02: Data from today already gathered. Validating.
06:26:03: The number of cards for date ID 22 is incorrect
06:26:03: Expected: 264    got: 250
06:26:03: Dataset invalid. Proceeding to run.
06:26:03: Update flag set to 1
07:43:06: Update flag set to 0
07:43:06: Message: Could not start a new session. New session request timed out
Build info: version: '4.0.0', revision: '3a21814679'
System info: host: '5168139481fc', ip: '192.168.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.16.3-microsoft-standard-WSL2', java.version: '11.0.11'
Driver info: driver.version: unknown
Stacktrace:
    at org.openqa.selenium.grid.sessionqueue.local.LocalNewSessionQueue.addToQueue (LocalNewSessionQueue.java:199)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.lambda$new$0 (NewSessionQueue.java:68)
    at org.openqa.selenium.remote.http.Route$TemplatizedRoute.handle (Route.java:192)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.execute (NewSessionQueue.java:120)
    at org.openqa.selenium.remote.tracing.SpanWrappedHttpHandler.execute (SpanWrappedHttpHandler.java:86)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.grid.router.Router.execute (Router.java:91)
    at org.openqa.selenium.grid.web.EnsureSpecCompliantResponseHeaders.lambda$apply$0 (EnsureSpecCompliantResponseHeaders.java:34)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$NestedRoute.handle (Route.java:270)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.AddWebDriverSpecHeaders.lambda$apply$0 (AddWebDriverSpecHeaders.java:35)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.netty.server.SeleniumHandler.lambda$channelRead0$0 (SeleniumHandler.java:44)
    at java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:515)
    at java.util.concurrent.FutureTask.run (FutureTask.java:264)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:829)

07:43:09: Data gathering run started.
07:43:09: Update flag set to 0
07:43:24: Data from today already gathered. Validating.
07:43:25: The number of cards for date ID 22 is incorrect
07:43:25: Expected: 264    got: 250
07:43:25: Dataset invalid. Proceeding to run.
07:43:25: Update flag set to 1
07:43:36: Update flag set to 0
07:43:36: Message: Reached error page: about:neterror?e=dnsNotFound&u=https%3A//www.cardmarket.com/en/Magic/Products/Singles/Battlebond%3Fsite%3D1&c=UTF-8&d=We%20can%E2%80%99t%20connect%20to%20the%20server%20at%20www.cardmarket.com.
Stacktrace:
WebDriverError@chrome://remote/content/shared/webdriver/Errors.jsm:181:5
UnknownError@chrome://remote/content/shared/webdriver/Errors.jsm:488:5
checkReadyState@chrome://remote/content/marionette/navigate.js:64:24
onNavigation@chrome://remote/content/marionette/navigate.js:312:39
emit@resource://gre/modules/EventEmitter.jsm:160:20
receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.jsm:42:25


07:43:39: Data gathering run started.
07:43:39: Update flag set to 0
07:43:54: Data from today already gathered. Validating.
07:43:55: The number of cards for date ID 22 is incorrect
07:43:55: Expected: 264    got: 250
07:43:55: Dataset invalid. Proceeding to run.
07:43:55: Update flag set to 1
07:48:55: Update flag set to 0
07:48:55: Message: Could not start a new session. New session request timed out
Build info: version: '4.0.0', revision: '3a21814679'
System info: host: '5168139481fc', ip: '192.168.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.16.3-microsoft-standard-WSL2', java.version: '11.0.11'
Driver info: driver.version: unknown
Stacktrace:
    at org.openqa.selenium.grid.sessionqueue.local.LocalNewSessionQueue.addToQueue (LocalNewSessionQueue.java:199)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.lambda$new$0 (NewSessionQueue.java:68)
    at org.openqa.selenium.remote.http.Route$TemplatizedRoute.handle (Route.java:192)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.grid.sessionqueue.NewSessionQueue.execute (NewSessionQueue.java:120)
    at org.openqa.selenium.remote.tracing.SpanWrappedHttpHandler.execute (SpanWrappedHttpHandler.java:86)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.grid.router.Router.execute (Router.java:91)
    at org.openqa.selenium.grid.web.EnsureSpecCompliantResponseHeaders.lambda$apply$0 (EnsureSpecCompliantResponseHeaders.java:34)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$NestedRoute.handle (Route.java:270)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.http.Route$CombinedRoute.handle (Route.java:336)
    at org.openqa.selenium.remote.http.Route.execute (Route.java:68)
    at org.openqa.selenium.remote.AddWebDriverSpecHeaders.lambda$apply$0 (AddWebDriverSpecHeaders.java:35)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.remote.ErrorFilter.lambda$apply$0 (ErrorFilter.java:44)
    at org.openqa.selenium.remote.http.Filter$1.execute (Filter.java:64)
    at org.openqa.selenium.netty.server.SeleniumHandler.lambda$channelRead0$0 (SeleniumHandler.java:44)
    at java.util.concurrent.Executors$RunnableAdapter.call (Executors.java:515)
    at java.util.concurrent.FutureTask.run (FutureTask.java:264)
    at java.util.concurrent.ThreadPoolExecutor.runWorker (ThreadPoolExecutor.java:1128)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run (ThreadPoolExecutor.java:628)
    at java.lang.Thread.run (Thread.java:829)

07:48:58: Data gathering run started.
07:48:58: Update flag set to 0
07:49:13: Data from today already gathered. Validating.
07:49:13: The number of cards for date ID 22 is incorrect
07:49:13: Expected: 264    got: 250
07:49:13: Dataset invalid. Proceeding to run.
07:49:13: Update flag set to 1
07:49:21: Update flag set to 0
07:49:21: Message: Reached error page: about:neterror?e=dnsNotFound&u=https%3A//www.cardmarket.com/en/Magic/Products/Singles/Battlebond%3Fsite%3D1&c=UTF-8&d=We%20can%E2%80%99t%20connect%20to%20the%20server%20at%20www.cardmarket.com.
Stacktrace:
WebDriverError@chrome://remote/content/shared/webdriver/Errors.jsm:181:5
UnknownError@chrome://remote/content/shared/webdriver/Errors.jsm:488:5
checkReadyState@chrome://remote/content/marionette/navigate.js:64:24
onNavigation@chrome://remote/content/marionette/navigate.js:312:39
emit@resource://gre/modules/EventEmitter.jsm:160:20
receiveMessage@chrome://remote/content/marionette/actors/MarionetteEventsParent.jsm:42:25


07:49:24: Data gathering run started.
07:49:24: Update flag set to 0
07:49:39: Data from today already gathered. Validating.
07:49:40: The number of cards for date ID 22 is incorrect
07:49:40: Expected: 264    got: 250
07:49:40: Dataset invalid. Proceeding to run.
07:49:40: Update flag set to 1
07:53:34: Update flag set to 0
07:53:34: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))

07:56:10: Data gathering run started.
07:56:10: Update flag set to 0
07:56:25: Data from today already gathered. Validating.
07:56:26: The number of cards for date ID 22 is incorrect
07:56:26: Expected: 264    got: 250
07:56:26: Dataset invalid. Proceeding to run.
07:56:26: Update flag set to 1
08:02:58: Update flag set to 0

08:02:58: Data gathering run started.
08:02:58: Update flag set to 0
08:03:13: Data from today already gathered. Validating.
08:03:14: Data validation complete.
08:03:14: Saving checksum: c0173c7248373d910780e0211361a71bad5b686f
08:03:15: Waiting for 1 hour.

08:13:30: Data gathering run started.
08:13:30: Update flag set to 0
08:13:30: Data from today already gathered. Validating.
08:13:31: Returning checksums: ['57b1125cf5034f7f9008b24b12e15dc7ce6741b4\n', '545aed86d25e1f908cab61a73ad86db55fc39a40\n', 'c0173c7248373d910780e0211361a71bad5b686f\n']
08:13:31: Data validation complete.
08:13:32: Saving checksum: c0173c7248373d910780e0211361a71bad5b686f
08:13:33: Waiting for 1 hour.

08:17:28: Data gathering run started.
08:17:28: Update flag set to 0
08:17:28: Data from today already gathered. Validating.
08:17:29: Returning checksums: ['57b1125cf5034f7f9008b24b12e15dc7ce6741b4', '545aed86d25e1f908cab61a73ad86db55fc39a40', 'c0173c7248373d910780e0211361a71bad5b686f', 'c0173c7248373d910780e0211361a71bad5b686f']
08:17:29: c0173c7248373d910780e0211361a71bad5b686f found in checksums
08:17:29: Dataset already validated.
08:17:29: Waiting for 1 hour.

08:33:30: Data gathering run started.
08:33:30: Update flag set to 0
08:33:30: Data from today already gathered. Validating.
08:33:30: Dataset already validated.
08:33:30: Waiting for 1 hour.

10:26:29: Data gathering run started.
10:26:29: Update flag set to 0
10:26:29: Data from today already gathered. Validating.
10:26:29: Dataset already validated.
10:26:29: Waiting for 1 hour.

10:30:03: Data gathering run started.
10:30:03: Update flag set to 0
10:30:03: Update flag set to 1

10:48:10: Data gathering run started.
10:48:10: Update flag set to 0
10:48:10: Update flag set to 1
10:48:50: Update flag set to 0
10:48:50: HTTPConnectionPool(host='firefox_webdriver', port=4444): Max retries exceeded with url: /wd/hub/session/d9e88098-046e-4e18-b817-ac2c0541656d/url (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f6841d57ca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))

10:50:24: Data gathering run started.
10:50:24: Update flag set to 0
10:50:24: Update flag set to 1
10:51:18: Update flag set to 0
10:51:18: module 'selenium.common.exceptions' has no attribute 'ErrorInResponseException'

21:23:53: Data gathering run started.
21:23:54: Data from today already gathered. Validating.
21:23:59: Dataset already validated.
21:23:59: Waiting for 1 hour.
22:23:59: Data from today already gathered. Validating.
22:23:59: Dataset already validated.
22:23:59: Waiting for 1 hour.
23:24:00: Data from today already gathered. Validating.
23:24:00: Dataset already validated.
23:24:00: Waiting for 1 hour.
00:24:00: Update flag set to 1
00:24:04: Update flag set to 0
00:24:04: Message: Unable to execute request for an existing session: Unable to find session with ID: 860d40ba-562f-4f9e-afad-20e54199204f
Build info: version: '4.0.0', revision: '3a21814679'
System info: host: '423089e0cba9', ip: '172.18.0.2', os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.16.3-microsoft-standard-WSL2', java.version: '11.0.11'
Driver info: driver.version: unknown

